src/test/cypress/integration/quiz-exercise/QuizExerciseAssessment.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { artemis } from '../../support/ArtemisTesting';
+import shortAnswerQuizTemplate from '../../fixtures/quiz_exercise_fixtures/shortAnswerQuiz_template.json';
+import multipleChoiceQuizTemplate from '../../fixtures/quiz_exercise_fixtures/multipleChoiceQuiz_template.json';
+
+// Accounts
+const admin = artemis.users.getAdmin();
+const tutor = artemis.users.getTutor();
+const student = artemis.users.getStudentOne();
+
+// Requests
+const courseManagementRequest = artemis.requests.courseManagement;
+
+// Common primitives
+let course: any;
+let quizExercise: any;
+
+describe('Quiz Exercise Assessment', () => {
+    before('Set up course', () => {
+        cy.login(admin);
+        courseManagementRequest.createCourse().then((response) => {
+            course = response.body;
+            courseManagementRequest.addStudentToCourse(course.id, student.username);
+            courseManagementRequest.addTutorToCourse(course, tutor);
+        });
+    });
+
+    afterEach('Delete Quiz', () => {
+        deleteQuiz();
+    });
+
+    after('Delete Course', () => {
+        cy.login(admin);
+        courseManagementRequest.deleteCourse(course.id);
+    });
+
+    describe('MC Quiz assessment', () => {
+        before('Creates a quiz and a submission', () => {
+            createQuiz();
+        });
+
+        it('Assesses a mc quiz submission automatically', () => {
+            cy.login(student);
+            courseManagementRequest.startExerciseParticipation(course.id, quizExercise.id);
+            courseManagementRequest.createMultipleChoiceSubmission(quizExercise, [0, 2]);
+            cy.wait(5000);
+            cy.visit('/courses/' + course.id + '/exercises/' + quizExercise.id);
+            cy.contains(quizExercise.title);
+            cy.contains('Score 50%');
+        });
+    });
+
+    describe('SA Quiz assessment', () => {
+        before('Creates a quiz and a submission', () => {
+            createQuiz(shortAnswerQuizTemplate);
+        });
+
+        it('Assesses a sa quiz submission automatically', () => {
+            cy.login(student);
+            courseManagementRequest.startExerciseParticipation(course.id, quizExercise.id);
+            courseManagementRequest.createShortAnswerSubmission(quizExercise, ['give', 'let', 'run', 'desert']);
+            cy.wait(5000);
+            cy.visit('/courses/' + course.id + '/exercises/' + quizExercise.id);
+            cy.contains(quizExercise.title);
+            cy.contains('Score 67%');
```

Comment:
I think you are missing assertions here. If you just want to verify the existence of the elements a `should('exist')` assertion should be added, but visibility is probably the better assertion.